src/test/java/in/projecteka/consentmanager/user/LockedUserServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package in.projecteka.consentmanager.user;
+
+import in.projecteka.consentmanager.user.model.LockedUser;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+class LockedUserServiceTest {
+
+    @Mock
+    LockedUsersRepository lockedUsersRepository;
+
+    @Mock
+    LockedServiceProperties lockedServiceProperties;
+
+    @BeforeEach
+    void init() {
+        initMocks(this);
+    }
+
+    @Test
+    void getLockedUser() {
+        var patientId = "patientId";
+        var lockedUser = new LockedUser(1, patientId, false, "","");
+        when(lockedUsersRepository.getLockedUserFor(patientId))
+                .thenReturn(Mono.just(lockedUser));
+        LockedUserService lockedUserService = new LockedUserService(lockedUsersRepository,lockedServiceProperties);
+        Mono<LockedUser> sessionPublisher = lockedUserService.userFor(patientId);
+
+        StepVerifier.create(sessionPublisher)
+                .assertNext(session -> assertThat(session).isEqualTo(lockedUser))
+                .verifyComplete();
+    }
+}
```

Comment:
verify(lockedUsersRepository, times(1)).getLockedUserFor(patientId);

This line verifies that the mock is called exactly for the number of times for the given method.

Reply:
done